analysis_test.go

```diff
@@ -434,19 +434,21 @@ func TestWorkmapToReach(t *testing.T) {
 	for name, fix := range table {
 		// Avoid erroneous errors by initializing the fixture's error map if
 		// needed
-		if fix.em == nil {
-			fix.em = make(map[string]*ProblemImportError)
-		}
+		t.Run(fmt.Sprintf("wmToReach(%q)", name), func(t *testing.T) {
```

Comment:
I think it's fine to just use the fixture name - it was only useful for providing context when the testing system itself didn't provide it directly. With subtests, that context is present.